front/components/PlansTables.tsx

```diff
@@ -390,3 +399,80 @@ export function PricePlans({
     );
   }
 }
+
+export function ProPricePlans({
```

Comment:
Let's make it a standard practice: one component per file 🙏.

Reply:
Can we first have a team discussion about it if you want to make it a standard practice? Cause I'm in favor of keeping things as is.